slam-ws/src/nav_rtab/package.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0"?>
```

Comment:
Similarly to the `CMakeLists`, remove unnecessary comments like we do with our other packages